indicator-stickynotes.py

```diff
@@ -256,8 +256,13 @@ def handler(indicator):
     #indicator.nset.notes[current_note].show()
     #indicator.nset.current_note = current_note
 
+def reload_handler(indicator):
+    # reload from data file on SIGUSR2
+    with open(os.path.expanduser(indicator.data_file), encoding="utf-8") as fsock:
+        indicator.nset.merge(fsock.read())
+    install_glib_handler(indicator, signal.SIGUSR2)
```

Comment:
@zkiss No I didn't think about that and yes it could cause data loss. The note being edited will be reset to the last saved version.
I think this problem can be avoided by checking the "last_modified" value so that only the newer notes will be merged.
I've also noticed that when my scripts successively creates new notes, some of them are missing. Scripts with short delays after each creation works fine. Do you have any idea how this could happen?